### PR TITLE
Add Znap package management

### PR DESCRIPTION
**Pull request by giard-alexandre**
- Add Znap Package Management
- Add optional Fig line

### CODE REVIEW

File: script/test [hunk 1]
```diff
@@ -15,15 +15,15 @@ git config --global user.email "dotfiles@demo.com"
...
+# load zshrc and check if znap is working
 echo "sourcing ~/.zshrc"
 # shellcheck disable=SC1090
 source ~/.zshrc || true
 
-test "$(antibody list | wc -l)" -gt 10 || {
-	echo "expecting 10+ bundles loaded" >&2
-	exit 1
-}
+# test "$(antibody list | wc -l)" -gt 10 || {
```
Comment (suarez-agu) [inline script/test:23]: Remove?

File: zsh/zshrc.symlink [hunk 2]
```diff
@@ -1,22 +1,34 @@
 #!/bin/zsh
+##
+# ⚠️ WARNING: Don't manually `source` your .zshrc file! It can have unexpected
```
Comment (suarez-agu) [inline zsh/zshrc.symlink:3]: Whyyyyyyy

**Review by suarez-agu — APPROVED** [review]

File: zsh/zshrc.symlink (hunk 2 above)
Comment (giard-alexandre) [reply zsh/zshrc.symlink:3]: Just DON'T :)

File: script/test (hunk 1 above)
Comment (giard-alexandre) [reply script/test:23]: unghhhhh

Comment (giard-alexandre) [reply script/test:23]: later xD

**Pull request merged**